test/controllers/nodes_controller_test.rb

```diff
@@ -6,23 +6,23 @@ class NodesControllerTest < ActionController::TestCase
   def test_routes
     assert_routing(
       { :path => "/api/0.6/node/create", :method => :put },
-      { :controller => "nodes", :action => "create" }
+      { :controller => "nodes", :action => "create", :format => "xml" }
     )
     assert_routing(
       { :path => "/api/0.6/node/1", :method => :get },
-      { :controller => "nodes", :action => "read", :id => "1" }
+      { :controller => "nodes", :action => "read", :id => "1", :format => "xml" }
     )
     assert_routing(
       { :path => "/api/0.6/node/1", :method => :put },
-      { :controller => "nodes", :action => "update", :id => "1" }
+      { :controller => "nodes", :action => "update", :id => "1", :format => "xml" }
     )
     assert_routing(
       { :path => "/api/0.6/node/1", :method => :delete },
-      { :controller => "nodes", :action => "delete", :id => "1" }
+      { :controller => "nodes", :action => "delete", :id => "1", :format => "xml" }
```

Comment:
PUT and DELETE related requests in this controller (as well as in the ways and relations controller) return plain text as a response (see https://github.com/openstreetmap/openstreetmap-website/blob/master/app/controllers/api/nodes_controller.rb#L54-L62).

I have some doubts if it's correct to assume format :xml in this case, and would only do so in case the response really uses XML format. Maybe I'm missing something here, though.

Reply:
Do those methods actually return anything at all?

Reply:
Just the version number, which isn't XML.

Oh hang on, this is an ancient and known-not-working PR of mine, so I'm not sure what your point is? I has assumed it was your XML rework...

Reply:
Oh, right, sorry for the confusion. It felt a bit silly to have so much duplication in my XML rework, started looking around, and came across this old PR. I was wondering wether that's something worth considering.